src/UI/Art/SpriteArtist/SpriteUIUtils.java

```diff
@@ -111,6 +111,8 @@ public static BufferedImage drawProseToWidth(String prose, int reqWidth)
       String line = lines.get(i);
       if( font.getWidth(line) <= reqWidth ) // if the line's short enough already, don't split it further
         continue;
+      if( i > 100 ) // bail; the text is too long or the space too small
+        return SpriteLibrary.createDefaultBlankSprite(1, 1);
```

Comment:
This is probably fine, but it wouldn't hurt to document more clearly why it's needed.